docs/topics/coroutine-context-and-dispatchers.md

```diff
@@ -246,9 +246,21 @@ fun main() {
 >
 {style="note"}
 
-It demonstrates several new techniques. One is using [runBlocking] with an explicitly specified context, and
-the other one is using the [withContext] function to change the context of a coroutine while still staying in the
-same coroutine, as you can see in the output below:
+The example above demonstrates two important techniques in coroutine usage.
+
+The first technique involves using [runBlocking] with an explicitly specified context.
+This ensures that the coroutine runs in a controlled environment,
+allowing you to define the execution context explicitly.
```

Comment:
I'm not sure what role this sentence plays. Yes, when you specify a context, the coroutine will run in that context. What are the other possible interpretations of `runBlocking(ctx1)`?

Reply:
My main issue with this sentence is that the phrase "controlled environment" don't seem to mean much. Does it imply that `runBlocking { }` is somehow an uncontrolled environment? Probably not. Does it say that we explicitly choose the execution context? No, the second half of the sentence does that. What is the role of this phrase, then? What do I understand when it's there that I wouldn't understand without it?